.github/workflows/validate-terraform.yml

```diff
@@ -0,0 +1,100 @@
+name: Website Checks
+on:
+  push:
+    branches:
+      - master
+      - 'release/**'
+  pull_request:
+    paths:
+      - .github/workflows/validate-terraform.yml
+      - website/docs/**
+
+jobs:
+  validate-terraform:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-pkg-mod-${{ hashFiles('go.sum') }}
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.14"
+      - name: install terrafmt
+        run: |
+          # go get github.com/katbyte/terrafmt
+          git clone --branch json-output --single-branch https://github.com/gdavison/terrafmt terrafmt
+          cd terrafmt
+          go install
+      - name: install tflint
+        env:
+          TFLINT_VERSION: "v0.18.0"
+        run: |
+          curl -s https://raw.githubusercontent.com/terraform-linters/tflint/master/install_linux.sh | sh
+      - name: lint code
```

Comment:
Let's consolidate this over to the existing `website.yml` 👍

Reply:
Yeah, I wasn't sure the best approach here, since we'll need to do an almost identical workflow for checking acceptance tests.

We're waiting on https://github.com/katbyte/terrafmt/pull/26 being merged for the supported release.